app/frontend/src/components/forms/ExportSubmissions.vue

```diff
@@ -168,11 +168,20 @@ export default {
               .utc()
               .format()
             : undefined;
+
+        // include deleted submissions in export. Default to false, but this could be set by user if implemented)
+        const showDeleted = false;
+
+        // include draft submissions in export. Default to false, but this could be set by user if implemented)
+        const showDrafts = false;
+
         const response = await formService.exportSubmissions(
           this.form.id,
           from,
           to,
-          this.exportFormat
+          this.exportFormat,
+          showDeleted,
+          showDrafts
         );
```

Comment:
Following up on the signature change proposal below, we can simplify this to the following:
```js
const response = await formService.exportSubmissions(
  this.form.id,
  this.exportFormat,
  {
    minDate: from,
    maxDate: to
    // showDeleted: undefined
    // showDrafts: undefined
  }
);
```
By having it as an options object as an argument, we can ensure that by default if a user calling this service function doesn't specify that option, it'll just default to false. As a bonus, this also makes the API call have less query parameters too simply by taking advantage of truthiness/falsiness JS behavior.